executor/src/witgen/query_processor.rs

```diff
@@ -135,7 +135,7 @@ where
                     }
                 } else {
                     Err(IncompleteCause::ExpressionEvaluationUnimplemented(
```

Comment:
BTW, I don't think we have an `IncompleteCause::ExpressionEvaluationUnimplemented`, because that is not a recoverable error, right? I would have found out what to do about my problem much quicker if there was just an `unimplemented!()` here!